spec/models/parking_registration_spec.rb

```diff
@@ -82,7 +82,7 @@
       registration = FactoryGirl.build(:parking_registration,
         spot_number: prev_registration.spot_number,
         parked_on: prev_registration.parked_on)
-      expect(registration.park).to be_false
+      expect(registration.park).to be(false)
       expect(registration).to_not be_valid
       expect(registration.errors[:spot_number]).to_not be_blank
 
```

Comment:
It would be great if we could update the syntax here. `should` is the old way of making assertions in rspec.

We should probably use `eq` instead of `be` wherever we can.